common/src/main/res/values/strings.xml

```diff
@@ -1114,4 +1114,9 @@
     <string name="only_favorites">Only Show Favorites</string>
     <string name="beacon_scanning">Beacon Monitor Scanning</string>
     <string name="car_data_unavailable">Open Home Assistant app to activate the sensor</string>
+    <string name="aa_set_favorites">Select your favorite entities to appear in the Favorites category in the Android Auto//Automotive app. You can also drag and drop to change the order in which they appear.</string>
+    <string name="aa_favorites">Android Auto Favorites</string>
+    <string name="aa_favorites_summary">Select your favorite entities to be shown in Android Auto</string>
+    <string name="android_automotive">Android Automotive</string>
+    <string name="android_automotive_favorites">Android Automotive Favorites</string>
```

Comment:
Still not sure about this; you're already using Automotive. Do we call the template UI Automotive mode vs native mode? Isn't native mode a feature of Automotive?

Reply:
Looking at the official docs the terms "Driving Optimized" is used a lot in both places. I don't think there is an exact name for it.

https://developer.android.com/training/cars

let me update the summary text to mention the screen shown while driving?

Reply:
ok that should hopefully be a bit more clear about what it is and where its used

Reply:
I don't see "Driving Optimized" on android.com/auto anywhere, the consumer-facing site.

> Driving Favorites

I like that, that should allow us to stay away from developer vocabulary.

Reply:
went with Driving Favorites here :)